modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -510,15 +510,51 @@ inline IE::Blob::Ptr extractRemoteBlob(IECallContext& ctx, std::size_t i) {
     using ParamType = std::pair<InferenceEngine::TensorDesc,
                                 InferenceEngine::ParamMap>;
 
-    ParamType* blob_params = cv::util::any_cast<ParamType>(&any_blob_params);
-    if (blob_params == nullptr) {
+    ParamType* frame_blob_params = cv::util::any_cast<ParamType>(&any_blob_params);
+    if (frame_blob_params == nullptr) {
         GAPI_Assert(false && "Incorrect type of blobParams: "
                               "expected std::pair<InferenceEngine::TensorDesc,"
                                                  "InferenceEngine::ParamMap>");
     }
 
-    return ctx.uu.rctx->CreateBlob(blob_params->first,
-                                   blob_params->second);
+    if (ctx.uu.params.device_id.find("GPU") != std::string::npos) {
+        GAPI_LOG_DEBUG(nullptr, "Extract remote blob for GPU DX11 device_id: " <<
+                                ctx.uu.params.device_id);
+        // despite of layout, blob dimensions always follow in N,C,H,W order
+        const auto& dims = frame_blob_params->first.getDims();
+        InferenceEngine::TensorDesc desc(InferenceEngine::Precision::U8,
+                                         {1, 1, dims[2], dims[3]},
+                                         InferenceEngine::Layout::NHWC);
+
+
+        InferenceEngine::ParamMap blobParams = {
+                           {"SHARED_MEM_TYPE", "VA_SURFACE"},
+                           {"DEV_OBJECT_HANDLE", frame_blob_params->second["DEV_OBJECT_HANDLE"]},
+                           {"VA_PLANE", uint32_t(0)}};
+        // TODO NV12 surface supported only
+        if (static_cast<InferenceEngine::ColorFormat>(frame_blob_params->second["COLOR_FORMAT"]) ==
+            InferenceEngine::ColorFormat::NV12) {
+            InferenceEngine::Blob::Ptr y_blob =
```

Comment:
Can be used `InferenceEngine::NV12Blob` for this case?

Reply:
it has huge class hierarchy, but actually final blob must be one of descendant of `ClBlob` which represents remote GPU memory. Maybe NV12Blob is present in class inheritance in the middle chain, i'm not sure